Packs/Malwarebytes/TestPlaybooks/Malwarebytes-Test.yml

```diff
@@ -0,0 +1,1169 @@
+            value:
+              simple: Malwarebytes.Endpoint.Assets
+            iscontext: true
+      - - operator: isExists
+          left:
+            value:
+              simple: Malwarebytes.Endpoint.Hostname
+            iscontext: true
+      - - operator: isExists
+          left:
+            value:
+              simple: Malwarebytes.Endpoint.IPAddress
+            iscontext: true
+      - - operator: isExists
+          left:
+            value:
+              simple: Malwarebytes.Endpoint.Domain
+            iscontext: true
+      - - operator: isExists
+          left:
+            value:
+              simple: Malwarebytes.Endpoint.MACAddress
+            iscontext: true
+      - - operator: isExists
+          left:
+            value:
+              simple: Malwarebytes.Endpoint.OS
+            iscontext: true
+      - - operator: isExists
+          left:
+            value:
+              simple: Malwarebytes.Endpoint.OSVersion
+            iscontext: true
+      - - operator: isExists
+          left:
+            value:
+              simple: Malwarebytes.Endpoint.Model
+            iscontext: true
+      - - operator: isExists
+          left:
+            value:
+              simple: Malwarebytes.Endpoint.Memory
+            iscontext: true
+      - - operator: isExists
+          left:
+            value:
+              simple: Endpoint.Hostname
+            iscontext: true
+      - - operator: isExists
+          left:
+            value:
+              simple: Endpoint.IPAddress
+            iscontext: true
+      - - operator: isExists
+          left:
+            value:
+              simple: Endpoint.Domain
+            iscontext: true
+      - - operator: isNotEmpty
+          left:
+            value:
+              simple: Endpoint.MACAddress
+            iscontext: true
+      - - operator: isExists
+          left:
+            value:
+              simple: Endpoint.OS
+            iscontext: true
+      - - operator: isExists
+          left:
+            value:
+              simple: Endpoint.OSVersion
+            iscontext: true
+      - - operator: isExists
+          left:
+            value:
+              simple: Endpoint.Model
+            iscontext: true
+      - - operator: isExists
+          left:
+            value:
+              simple: Endpoint.Memory
+            iscontext: true
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 3800
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+  "20":
+    id: "20"
+    taskid: 7366c22e-12e4-4207-8cbc-420aee8bf588
+    type: regular
+    task:
+      id: 7366c22e-12e4-4207-8cbc-420aee8bf588
+      version: -1
+      name: malwarebytes-get-scan-detections
+      script: '|||malwarebytes-get-scan-detections'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "21"
+    scriptarguments:
+      job_id:
+        simple: ${inputs.job_id}
```

Comment:
why is this an input? cant we get it from previous tasks? the test playbook will never be able to run